doc/win-dev.ps1

```diff
@@ -70,43 +70,26 @@ try {
 	$Env:Path += $ChocoPath
 }
 
-
-choco install -y "visualstudio${VsVersion}community"
-ThrowOnNativeFailure
-
-choco install -y "visualstudio${VsVersion}-workload-netcoretools"
-ThrowOnNativeFailure
-
-choco install -y "visualstudio${VsVersion}-workload-vctools"
-ThrowOnNativeFailure
-
-choco install -y "visualstudio${VsVersion}-workload-manageddesktop"
-ThrowOnNativeFailure
-
-choco install -y "visualstudio${VsVersion}-workload-nativedesktop"
-ThrowOnNativeFailure
-
-choco install -y "visualstudio${VsVersion}-workload-universal"
-ThrowOnNativeFailure
-
-choco install -y "visualstudio${VsVersion}buildtools"
-ThrowOnNativeFailure
-
-
-choco install -y git
-ThrowOnNativeFailure
-
-choco install -y cmake
-ThrowOnNativeFailure
-
-choco install -y winflexbison3
-ThrowOnNativeFailure
-
-choco install -y windows-sdk-8.1
-ThrowOnNativeFailure
-
-choco install -y wixtoolset
-ThrowOnNativeFailure
+# GitHub Actions uses an image that comes with most dependencies preinstalled. Don't install them twice.
+if (-not $Env:GITHUB_ACTIONS) {
```

Comment:
One of the things I dislike in every p/l: if not, else. I prefer the inverted condition with swapped branches.

Reply:
My reasoning for doing it this way around: As the script is in `doc/`, I consider the installation on a fresh system its primary use so I handled that first.